website/docs/tutorials/atmos-getting-started.md

```diff
@@ -201,8 +238,12 @@ workflows:
       - command: terraform deploy output-results
 ```
 
-Above we updated a couple variables to change the behavior of our terraform code for this particular stack. Since we mounted our local `stacks/` folder to our `atmos` container via `--volume`, when you save the above stack file docker will update your container's `/stacks/example.yaml` file as well. Now to execute this again... we simply invoke our `deploy-all` workflow command:
-This should run through our workflow similar to the way we did it before. Still, this time we'll see our temperature return from the weather API for the date you specified instead of today's date. We'll skip over our terraform `local-exec`'s `echo` command for "pretty printing" our weather data. Instead, we'll just get our updated weather information as one of our `Outputs`.
+Above we updated a couple variables to change the behavior of our terraform code for this particular stack. Since we mounted our local `stacks/`
```

Comment:
```suggestion
Above, we updated a couple of variables to change the behavior of our terraform code for this particular stack. Since we mounted our local `stacks/`
```